tests/integration/test_http_transport.py

```
@@ -0,0 +1,83 @@
+    env = os.environ.copy()
+    env.update({
+        "MCP_TRANSPORT": "http",
+        "MCP_HOST": "127.0.0.1",
+        "MCP_PORT": "8765",
+        "ZAMMAD_URL": "http://mock.zammad.com/api/v1",
+        "ZAMMAD_HTTP_TOKEN": "test-token",
+    })
+
+    # Start server process
+    process = subprocess.Popen(
+        ["python", "-m", "mcp_zammad"],
```

Ruff (0.14.4)

Starting a process with a partial executable path

(S607)